source/security/index.html.md.erb

```diff
@@ -31,40 +41,70 @@ To further secure your live developer keys:
 
 ## Securing your integration with GOV.UK Pay
 
-Make sure you’ve fully tested your integration with GOV.UK Pay. When doing so, take care not to use any real card numbers. Read our testing section for more details.
+Make sure you’ve fully tested your integration with GOV.UK Pay. When doing so,
+take care not to use any real card numbers. Read the
+[_Testing__](/testing_govuk_pay) section for more details.
 
 ## Securing user information
 
-GOV.UK Pay does not store full card numbers or CVV data for security reasons. This means you will not be able to search for transactions using card numbers.
+GOV.UK Pay does not store full card numbers or CVV data for security reasons.
+This means you will not be able to search for transactions using card numbers.
 
 ## Payment Card Industry (PCI) compliance
 
-Anyone involved with the processing, transmission, or storage of cardholder data must comply with the [Payment Card Industry Data Security Standards](https://www.pcisecuritystandards.org/) (PCI DSS) [external link]. GOV.UK Pay is certified as fully compliant as a Level 1 Service Provider with PCI DSS version 3.2. All GOV.UK Pay partners must be compliant with PCI DSS, and must validate their compliance annually.
+Anyone involved with the processing, transmission, or storage of cardholder
+data must comply with the [Payment Card Industry Data Security
+Standards](https://www.pcisecuritystandards.org/) (PCI DSS) [external link].
+GOV.UK Pay is certified as fully compliant as a Level 1 Service Provider with
+PCI DSS version 3.2. All GOV.UK Pay partners must be compliant with PCI DSS,
+and must validate their compliance annually.
 
 ### Use your Merchant ID to report PCI DSS compliance
 
-A merchant ID is a unique number that identifies you to your payment processor and acquiring bank. The recommended approach is to report PCI DSS compliance by MID by:
+A merchant ID is a unique number that identifies you to your payment processor
+and acquiring bank. The recommended approach is to report PCI DSS compliance
+by MID by:
 
-- agreeing with your acquiring bank to allocate an unique MID for each separate payment channel you have in place
-- reporting your PCI DSS compliance status for each of these unique MIDs to your acquiring bank
+* agreeing with your acquiring bank to allocate an unique MID for each
+separate payment channel you have in place
+* reporting your PCI DSS compliance status for each of these unique MIDs to
+your acquiring bank
 
-If you have one MID that encompasses multiple payment channels, the compliance requirements will be more complex for that MID. Check the PCI DSS for more information.
+If you have one MID that encompasses multiple payment channels, the compliance
+requirements will be more complex for that MID. Check the PCI DSS for more
```

Comment:
Where would be the best place to link to here?

Reply:
i can't find a specific page for MIDs, plus we probably don't want to link to a URL that could change? if that's the case, we could use the homepage https://www.pcisecuritystandards.org/

Reply:
If you think it's best we don't link to anywhere here, that's fine too

Reply:
i think it's better to push them to the PCI site to find the information they need